theme/themes/pxt/globals/site.variables

```diff
@@ -160,6 +161,17 @@
 @trashIconColor: @primaryColor;
 
 /*-------------------
+   Serial editor
+--------------------*/
+@serialBackgroundColor: lighten(@simulatorBackground, 2);
+@serialTextColor: white;
+@serialGraphBackground: @lightGrey;
+@serialGraphColor: black;
+@closeColor: black;
```

Comment:
rename to editorCloseColor. This will conflict with semantic's close color here.